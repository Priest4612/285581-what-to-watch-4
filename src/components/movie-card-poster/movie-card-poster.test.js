import React from 'react';
import renderer from 'react-test-renderer';

import MovieCardPoster from './movie-card-poster.jsx';


it(`Render MovieCardPoster -- full`, () => {
  const tree = renderer
    .create(<MovieCardPoster
      name={`name-poster`}
      posterImage={`posterImage`}
      isFull={true}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});


it(`Render MovieCardPoster  -- main`, () => {
  const tree = renderer
    .create(<MovieCardPoster
      name={`name-poster`}
      posterImage={`posterImage`}
      isFull={false}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
