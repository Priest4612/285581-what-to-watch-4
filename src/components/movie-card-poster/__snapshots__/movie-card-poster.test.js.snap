// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieCardPoster  -- main 1`] = `
<div
  className="movie-card__poster
        false"
>
  <img
    alt="name-poster"
    height="327"
    src="posterImage"
    width="218"
  />
</div>
`;

exports[`Render MovieCardPoster -- full 1`] = `
<div
  className="movie-card__poster
        movie-card__poster--big"
>
  <img
    alt="name-poster"
    height="327"
    src="posterImage"
    width="218"
  />
</div>
`;
